src/site/content/en/blog/state-of-css-2022/index.md

```diff
@@ -1438,7 +1438,7 @@ articulate over and under conditions. It may look like this:
 After media query ranges, the same media query could look like this:
 
 ```css
-@media (320px >= width) {
+@media (width >= 320px) {
```

Comment:
nice catch, thank you! this update makes the query match the min-width version 👍🏻

Reply:
thanks for the amazing article!